packages/components-sketch/views/button.hbs

```diff
@@ -9,313 +9,655 @@ file, You can obtain one at https://mozilla.org/MPL/2.0/.
-        </div>
-        <div class="agent-states--item">
-            <p class="agent-states--label">:focus</p>
-            <scale-button data-sketch-key="button-7" variant="primary" data-sketch-state="focus"><scale-icon-action-search></scale-icon-action-search>Button Label</scale-button>
-        </div>
-        <div class="agent-states--item">
-            <p class="agent-states--label">:active/:selected</p>
-            <scale-button data-sketch-key="button-8" variant="primary" data-sketch-state="active"><scale-icon-action-search></scale-icon-action-search>Button Label</scale-button>
-        </div>
-        <div class="agent-states--item">
-            <p class="agent-states--label">:disabled</p>
-            <scale-button data-sketch-key="button-9" variant="primary" disabled="true"><scale-icon-action-search></scale-icon-action-search>Button Label</scale-button>
-        </div>
-    </div>
-    
-    <h2 class="title--artboard-section">Primary – Icon only</h2>
-    <div class="agent-states--grid" data-category="Icon only">
-        <div class="agent-states--item">
-            <p class="agent-states--label">:enabled</p>
-            <scale-button data-sketch-key="button-10" variant="primary" icon-only="true"><scale-icon-action-search></scale-icon-action-search></scale-button>
-        </div>
-        <div class="agent-states--item">
-            <p class="agent-states--label">:hover</p>
-            <scale-button data-sketch-key="button-11" variant="primary" icon-only="true" data-sketch-state="hover"><scale-icon-action-search></scale-icon-action-search></scale-button>
-        </div>
-        <div class="agent-states--item">
-            <p class="agent-states--label">:focus</p>
-            <scale-button data-sketch-key="button-12" variant="primary" icon-only="true" data-sketch-state="focus"><scale-icon-action-search></scale-icon-action-search></scale-button>
-        </div>
-        <div class="agent-states--item">
-            <p class="agent-states--label">:active/:selected</p>
-            <scale-button data-sketch-key="button-13" variant="primary" icon-only="true" data-sketch-state="active"><scale-icon-action-search></scale-icon-action-search></scale-button>
-        </div>
-        <div class="agent-states--item">
-            <p class="agent-states--label">:disabled</p>
-            <scale-button data-sketch-key="button-14" variant="primary" icon-only="true" disabled="true"><scale-icon-action-search></scale-icon-action-search></scale-button>
-        </div>
+  <h1 class="title--artboard">Button Large</h1>
+
+  <h2 class="title--artboard-section">Primary – Text Only</h2>
+  <div class="agent-states--grid" data-category="Text only">
+    <div class="agent-states--item">
+      <p class="agent-states--label">:enabled</p>
+      <scale-button data-sketch-key="button-0" variant="primary"
+        >Button Label</scale-button
+      >
```

Comment:
this formatting is odd! prettier does this?

Reply:
yas 🌝

Reply:
we could tell prettier to alter the html formatting maybe 🤔

Reply:
im fine with ugly prettier output 🤞🏻